Code/Framework/AzCore/Tests/Math/QuaternionTests.cpp

```diff
@@ -510,4 +510,16 @@ namespace UnitTest
     }
 
     INSTANTIATE_TEST_CASE_P(MATH_Quaternion, QuaternionScaledAxisAngleConversionFixture, ::testing::ValuesIn(RotationRepresentationConversionTestQuats));
+
+    TEST(MATH_Quaternion, ShortestEquivalent)
+    {
+        const AZ::Quaternion testQuat = AZ::Quaternion::CreateRotationX(AZ::Constants::HalfPi * 3.0f);
+
+        AZ::Quaternion absQuat = testQuat;
+        absQuat.ShortestEquivalent();
+        EXPECT_TRUE(absQuat.IsClose(testQuat.GetShortestEquivalent(), 1e-6f));
```

Comment:
There's a matcher defined in `Code/Framework/AzCore/Tests/AZTestShared/Math/MathTestHelpers.h` that is better for this, it gives more detailed information when the test fails:
```suggestion
        EXPECT_THAT(testQuat, IsCloseTolerance(absQuat, 1e-6f));
```

Reply:
Good one, changed.